models/public/mozilla-deepspeech-0.8.2/mozilla-deepspeech-0.8.2.md

```diff
@@ -0,0 +1,117 @@
+# mozilla-deepspeech-0.8.2
+
+## Use Case and High-Level Description
+
+"mozilla-deepspeech-0.8.2" is a speech recognition neural network pre-trained by Mozilla
+based on DeepSpeech architecture (CTC decoder with beam search and n-gram language model)
+with changed neural network topology.
+
+For details on the original DeepSpeech, see paper <https://arxiv.org/abs/1412.5567>.
+
+For details on this model, see <https://github.com/mozilla/DeepSpeech/releases/tag/v0.8.2>.
+
+## Specification
+
+| Metric                          | Value                                     |
+|---------------------------------|-------------------------------------------|
+| Type                            | Speech recognition                        |
+| GFlops per audio frame          | 0.0472                                    |
+| GFlops per second of audio      | 2.36                                      |
+| MParams                         | 47.2                                      |
+| Source framework                | TensorFlow\*                              |
+
+## Accuracy
+
+| Metric                       | Value      | Parameters                                                       |
+| ---------------------------- | ---------- | ---------------------------------------------------------------- |
+| WER @ LibriSpeech test-clean | 8.39%      | with LM, beam_width = 32, Python CTC decoder, accuracy checker   |
+| WER @ LibriSpeech test-clean | **6.13%**  | with LM, **beam_width = 500**, C++ CTC decoder, accuracy checker |
+| WER @ LibriSpeech test-clean | **6.15%**  | with LM, **beam_width = 500**, C++ CTC decoder, demo             |
+
+*NB*: beam_width=32 is a low value for a CTC decoder, and was used to achieve reasonable evaluation time with Python CTC decoder in Accuracy Checker.
+Increasing beam_width improves WER metric and slows down decoding.  Speech recognition demo has a faster C++ CTC decoder module.
+
+## Performance
```

Comment:
I thought we already removed empty sections from demo's readme, but it still be there

Reply:
OK, I'll fix it now.

Reply:
Done.

Reply:
CI pipeline error is not mine -- it's "broken" link to AlexNet paper.